.github/workflows/auto-readme.yml

```diff
@@ -15,21 +17,35 @@ on:
 
 jobs:
   update:
-    if: github.event_name == 'schedule'
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
 
+    - name: Find default branch name
+      id: defaultBranch
+      shell: bash
+      env:
+        GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      run: |
+        default_branch=$(gh repo view --json defaultBranchRef --jq .defaultBranchRef.name)
+        printf "::set-output name=defaultBranch::%s\n" "${default_branch}"
+        printf "defaultBranchRef.name=%s\n" "${default_branch}"
```

Comment:
I think you can get this with `${{ github.event.repository.default_branch }}`

Reply:
Haha, awesome. I didn't expect this to show up on this PR, but thanks for the feedback and I'll improve it.

Reply:
@mcalhoun @dylanbannon `github.event.repository.default_branch` is not available when, as in this workflow, the `event` is `schedule`.